flexget/plugins/cli/series.py

```diff
@@ -109,7 +109,10 @@ def display_summary(options):
         console('ERROR: %s' % str(e))
         return
     if not porcelain:
-        console(footer)
+        if query.count() == 0:
```

Comment:
Excellent!
I was a bit reluctant to change too much but I considered doing just that. Will do.